_includes/post-preview.html

```diff
@@ -0,0 +1,31 @@
+{% assign post = include.post %}
+
+{%- capture excerpt_link_url -%}
+  {%- if post.excerpt_link_url -%}
+    {{ post.excerpt_link_url }}
+  {%- else -%}
+    {{ site.baseurl }}{{ post.url }}
+  {%- endif -%}
+{%- endcapture -%}
+
+<article id="{{ post.id | default: post.title | slugify }}" class="post-preview">
+  <header class="post-preview__header">
+    <p class="post-category site-subheading">
+      {{ post.tags[0] }}
```

Comment:
**Question:** I noticed some posts have multiple tags. Would there be a reason we want to display multiple?

Also, if we're only relying on the first tab for these previews, do we have guidance on what should be listed as the first tag when a post has multiple?

Reply:
This is something I want to explore in phase 2 of this layout (issue #2967), particularly when we get into exploring filtering the content via content types and categories. We broke this into very small incremental pieces for now, but will be sure to touch on how tags are handled soon! 

Recommend we keep this as is for now and explore additional enhancements in #2967.